tests/integration/commands/dev/dev-miscellaneous.test.ts

```diff
@@ -633,7 +633,7 @@ describe.concurrent('commands/dev-miscellaneous', () => {
         .withEdgeFunction({
           config: { onError: 'bypass', path: '/hello-1' },
           handler: () => {
-            // @ts-expect-error: Intentionally referencing an undefined global
+            // @ts-expect-error TS(2304) FIXME: Cannot find name 'ermThisWillFail'.
             // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
These are intentional errors. You could just pass in the handler as a string here to avoid the type and eslint errors.

Reply:
Fixed